pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[build-system]
+requires = ["setuptools>=61.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "hapkert"
+version = "0.1.0"
+description = "A Python package for Hapke radiative transfer calculations"
+readme = "README.md"
+requires-python = ">=3.6"
+license = { text = "MIT" }
+authors = [
+    { name = "Ryleigh Davis", email = "rdavis@caltech.edu" }
+]
+dependencies = [
+    "numpy",
+    "scipy",
+    "astropy",
+    "miepython",
+    "pandas",
```

Comment:
You originally just had `numpy` listed, but it looked like you were using all of the above. In general you can just test this out by creating a fresh virtual environment (as described in `README.md`) and then seeing whether a `pip install` of your hapkert package is enough to actually run the example code. When you `pip install`, it uses the dependencies listed here.